package.json

```diff
@@ -4,7 +4,7 @@
   "description": "Vue library for input formats",
   "main": "./src/index.js",
   "scripts": {
-    "test": "jest",
+    "test": "jest --config jest.config.js",
```

Comment:
run jest from config

Reply:
By default, jest will search for nearest config, isn't it ?